lib/namespace.js

```diff
@@ -0,0 +1,13 @@
+const uuid = require('uuid/v4');
+const { createNamespace } = require('cls-hooked');
+
+const ns = createNamespace(`express-wolox-logger:${uuid()}`);
+
+const getRequestId = () => (ns ? ns.get('requestId') : null);
+
+const setRequestId = (id = uuid()) => {
```

Comment:
maybe we could use smaller ids. 
Also in future version we should make this customizable, like find a way to let each project provide a custom random id generator function.